### PR TITLE
Chore : Introduce logrus and setup log level depending on flags

### PR DESCRIPTION
**Pull request by seblegall**
The objectif here is to use logs (through logrus) instead of printing outputs on the standard output using the fmt package.

### CODE REVIEW

File: pkg/kubernetes/namespace.go [hunk 1]
```diff
@@ -90,13 +89,13 @@ func (ns *namespaceRepository) WatchPhase(emit resource.EventEmitter) error {
...
 	if err != nil {
-		log.Printf("[WATCHER] %s", err.Error())
+		logrus.Errorf("Error when watching phase : %s", err.Error())
 		return err
 	}
 
 	defer func() {
 		watcher.Stop()
-		log.Printf("[WATCHER] restart watcher due to connection close")
+		logrus.Info("restarting pahse watcher due to connection close")
```
Comment (pboiseau) [inline pkg/kubernetes/namespace.go]: typo restarting ***pahse*** watcher due to connection close

File: pkg/resource/namespace.go [hunk 2]
```diff
@@ -154,6 +158,21 @@ func (ns *namespaceService) watchStatus() error {
...
+	for _, e := range before {
+		s1 = append(s1, e.Namespace)
+	}
+	logrus.Debugf("before : %s", s1)
+
+	logrus.Debug("after")
+	for _, e := range now {
+		s2 = append(s2, e.Namespace)
+	}
+	logrus.Debugf("after : %s", s2)
```
Comment (pboiseau) [inline pkg/resource/namespace.go]: remove this because this was only for debugging ?

**Comment by pboiseau** [comment]
Don't forget to update https://github.com/Meetic/blackbeard/blob/feat/add-more-logs/docs/workflow/cli.md documentation to add `-v` for log.

File: pkg/kubernetes/namespace.go (hunk 1 above)
Comment (seblegall) [reply pkg/kubernetes/namespace.go]: deleted logs due to refactoring on previous branch.

File: pkg/resource/namespace.go (hunk 2 above)
Comment (seblegall) [reply pkg/resource/namespace.go]: yep.

**Pull request merged**